### ISSUE TITLE
Could we add an option into the schema to disable/enable whole/part of an editor?

### ISSUE THREAD

**Issue by ricklxm** [jdorn/json-editor#430]
Scenario:
I'm review an application. The application is load from server and displayed as a form. 
Besides the information of this application, there is another checkbox ['Pass','Fail'] for setting the status of  an application. Maybe there is a textarea to write the reason. So I want the information part of this form is readonly, and the rest part is writable.

**Comment by leeorengel** [comment]
You can do this with disable() on any editor. E.g.

`editor.getEditor('root.location').disable();`

**Comment by ricklxm** [comment]
@leeorengel, yes, you are right. I post this issue here is because I think it will make things much more easier if there is a built-in option for this in the schema.

**Comment by yiwenliuable** [comment]
+1
